parsers/lib/validation.py

```diff
@@ -71,8 +71,8 @@ def validate(datapoint, **kwargs):
 
     if remove_negative:
         for key, val in generation.items():
-            if val is not None and val < 0.0:
-                print("{} returned negative value for {}".format(key,datapoint['countryCode']))
+            if val is not None and -5.0 < val < 0.0:
+                print("{} returned {} for {}, setting to None".format(key, val, datapoint['zoneKey']))
```

Comment:
can we use `logger.warn` to log the warning?

Reply:
sure I will do the same for the other prints, do I put logger=None in the function args?